.buildkite/scripts/test.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+set -euxo pipefail
```

Comment:
Could this script launch `.ci/test.sh`, it looks like this script is doing more or less the same as that other file.

Or maybe, could we use that script directly as a command in the step definition ?